test/index.js

```diff
@@ -129,11 +155,21 @@ async function test() {
 }
 
 async function tryStuff() {
-    var runner = await deployIfNeeded()
+    var ret =  await deployIfNeeded()
```

Comment:
You can use `var [runner, verifier] = await deployIfNeeded()` here

Reply:
ah ok. now i know . thanks.